### PR TITLE
Configure JCommander to accept unparsed flags.

### PR DESCRIPTION
**Pull request by moribellamy**
Need this for a hermetic deployment in which we bundle a JRE.

### CODE REVIEW

File: proxy/src/main/java/com/wavefront/agent/AbstractAgent.java [hunk 1]
```diff
@@ -240,7 +245,9 @@ private void loadListenerConfigurationFile() throws IOException {
    */
   public void start(String[] args) throws IOException {
     try {
+      logger.info("Arguments: " + Joiner.on(", ").join(args));
       new JCommander(this, args);
+      logger.info("Unparsed arguments: " + Joiner.on(", ").join(unparsed_params));
```
Comment (panghy) [inline proxy/src/main/java/com/wavefront/agent/AbstractAgent.java]: can unparsed_params be null?

Comment (moribellamy) [inline proxy/src/main/java/com/wavefront/agent/AbstractAgent.java]: Yes, thanks. FIxed.

**Comment by panghy** [comment]
Please rebase from master

**Comment by moribellamy** [comment]
Done

**Pull request merged**